### DISCUSSION TITLE
API information

### DISCUSSION THREAD

**Discussion by zantler** [TypedProject/ts-express-decorators#1603] [Q&A] [answered]
Hi guys,
Anypne knows where to find the log for the APIs information? 
![image](https://user-images.githubusercontent.com/89181258/138717460-412c8866-e81a-492f-a889-fef0d1ebe81f.png)
Im using the version 5.X

Thanks for your help!

**Comment by Romakita** [comment, marked as answer]
What you mean by logs for Api information?
Swagger? Routes mounted by Ts.ED?

**Reply by zantler** [reply]
Each time i call an API on my app, that log shows up on console. I need to know where i can find the code for that log cause i need to catch the resposne information and the afterRoutesInit is not working

**Reply by Romakita** [reply]
Ok I see. In v5 you can override the LogIncomingRequestMiddleware ;)

https://github.com/tsedio/tsed/blob/5.x/packages/common/src/platform/middlewares/LogIncomingRequestMiddleware.ts

**Reply by zantler** [reply]
Thanks a lot!!
